docs/conf.py

```diff
@@ -39,8 +39,10 @@
 os.environ['QISKIT_DOCS'] = 'TRUE'
 
 # -- Project information -----------------------------------------------------
+import datetime
```

Comment:
I'd probably group this with the other imports just for simplicity.

Reply:
Done in [abeb9d7](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1166/commits/abeb9d7c6a66569b435233b6c27f51df1cca84b7)